core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -1,6 +1,7 @@
 <div  class="oppia-drop-area" ng-if="showDropArea">
   <div class="oppia-blur-background"></div>
-  <div class="oppia-drop-area-message">Drop your file here!</div>
+  <div class="oppia-drop-area-message" ng-if="guest == false && disallowed == false">Drop your file here!</div>
+  <div class="oppia-drop-area-message-disallow" ng-if="guest">Please log in to upload audio files.</div>
```

Comment:
Give a better name to class like `oppia-drop-area-message-for-guest-users`

Reply:
Should be done.